wrappers/python/nntile/utils/constructors.py

```diff
@@ -0,0 +1,55 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/pipeline.py
+# TRaining pipeline of NNTile Python package
+#
+# @version 1.0.0
+
+from nntile.nntile_core.tensor import TensorTraits, Tensor_fp32, Tensor_fp64, \
+        Tensor_int64, Tensor_fp16, Tensor_bool
+from nntile.functions import fill_async
+import numpy as np
+
+nnt2np_type_mapping = {
+    Tensor_fp32: np.float32,
+    Tensor_fp64: np.float64
+}
+
+np2nnt_type_mapping = {
+    np.float32: Tensor_fp32,
+    np.float64: Tensor_fp64
+}
+
+
+
+def tensor_from_numpy(A: np.array, basetile_shape: tuple = None, mpi_distr = [0], next_tag = 0):
```

Comment:
It better to name it in NumPy-like way. Something like `astensor` or just `tensor`.

Reply:
Made "astensor" 
Just tensor will get confused with internal tensor and tensor.py